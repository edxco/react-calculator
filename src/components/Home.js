import React from 'react';

function Home() {
  return (
    <div className="Home px-5">
      <h1>Changelog</h1>
      <p className="text-regular changelog-h1">
        All notable changes to this project will be documented in this file.
        <br />
        [1.0.0] - 2021-07-02
        <br />
        <br />
        <strong>Added</strong>
        <br />
        - New visual identity
        <br />
        - Version navigation
        <br />
        - Link to changelog add
        <br />
      </p>
    </div>
  );
}

export default Home;
