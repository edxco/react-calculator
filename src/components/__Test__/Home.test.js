import React from 'react';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Home from '../Home';

test('return content from Home matches snapshot', () => {
  const content = render(<Home />);
  expect(screen.getByText('Changelog')).toBeInTheDocument();
});
