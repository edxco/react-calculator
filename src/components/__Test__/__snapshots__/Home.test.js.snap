// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`return content from Home matches snapshot 1`] = `
<div>
  <div
    class="Home"
  >
    <h1>
      Welcome to our page
    </h1>
    <p>
      Lorem ipsum odor amet, consectetuer adipiscing elit. Ante metus hac phasellus himenaeos potenti; arcu gravida fermentum? Pharetra purus nec parturient placerat hendrerit tortor. Inceptos quisque consequat lobortis; fringilla tempor risus. Ante at velit nullam elit nascetur condimentum class nostra sociosqu. Sodales odio ut dis morbi convallis fringilla. Ut ornare massa ullamcorper senectus elementum. Non habitant purus vestibulum vel curabitur iaculis elementum. Senectus ipsum non mus natoque netus netus fringilla, ut morbi. Risus orci adipiscing risus odio gravida in.
    </p>
    <p>
      Aliquet nostra tristique dictum consequat congue morbi dolor nascetur. Montes vehicula erat nisi; pretium convallis aptent porttitor odio. Nascetur tortor iaculis metus fermentum potenti habitasse. Facilisi nullam amet iaculis orci cursus rutrum est penatibus. Fringilla tellus efficitur per odio pharetra velit pharetra. Dapibus eget proin curabitur conubia potenti.
    </p>
  </div>
</div>
`;
